components/KebabPopUp.tsx

```diff
@@ -44,44 +52,48 @@ export const KebabPopUp: React.FC<KebabPopUpProps> = props => {
           />
         </Row>
         <ScrollView>
-          <ListItem bottomDivider>
-            <ListItem.Content>
-              <ListItem.Title
-                onPress={controller.PIN_CARD}
-                {...testIDProps('pinOrUnPinCard')}>
-                <Text size="small" weight="bold">
-                  {props.vcMetadata.isPinned ? t('unPinCard') : t('pinCard')}
-                </Text>
-              </ListItem.Title>
-            </ListItem.Content>
-          </ListItem>
-
-          <WalletBinding
-            label={t('offlineAuthenticationDisabled!')}
-            content={t('offlineAuthDisabledMessage')}
+          <KebabPopupListItemContainer
+            label={props.vcMetadata.isPinned ? t('unPinCard') : t('pinCard')}
+            listItemIcon={SvgImage.OutlinedPinIcon()}
+            onPress={controller.PIN_CARD}
+            testID="pinOrUnPinCard"
+          />
+          <ShareVc
+            testID="shareVcFromKebab"
+            label={t('share')}
             service={props.service}
-            vcMetadata={props.vcMetadata}
+            flowType={VCShareFlowType.MINI_VIEW_SHARE}
           />
-
+          {props.vcHasImage ? (
```

Comment:
This ternary check can be modified to conditional check

props.vcHasImage  && <>